x-pack/docs/en/security/securing-communications/security-minimal-setup.asciidoc

```diff
@@ -0,0 +1,168 @@
+[[security-minimal-setup]]
+=== Set up minimal security for {es}
+++++
+<titleabbrev>Set up minimal security</titleabbrev>
+++++
+
+In {es} 8.0 and later, security is
+<<configuring-stack-security,enabled automatically>> when you start {es} for the 
+first time. 
+
+If you're running an existing {es} cluster where security is disabled, you can 
+manually enable the {es} {security-features} and then create passwords for 
+built-in users. You can add more users later, but using the built-in users 
+simplifies the process of enabling security for your cluster.
+
+include::../security-manual-configuration.asciidoc[tag=minimal-security-note]
+
+The basic license includes minimal security settings for the {stack}, so
```

Comment:
Using `minimal` here is a bit confusing: This page's title is about `minimal security`. It is the least amount of security that a user can enable for some protection. But It is definitely *not* what a basic license offers, which is much more than `minimal`.

Reply:
I think that this section about licenses doesn't make sense here anymore. I'm going to remove it.